src/Orchard.Web/Modules/Orchard.OutputCache/Filters/OutputCacheFilter.cs

```diff
@@ -370,6 +370,9 @@ public class OutputCacheFilter : FilterProvider, IActionFilter, IResultFilter, I
             foreach (var p in filterContext.ActionParameters)
                 result.Add("PARAM:" + p.Key, p.Value);
 
+            // Vary by scheme (See #6850)
```

Comment:
Remove the issue reference and end the comment in a period (like the one below). I don't think we use the practice of referencing GitHub issues from code (but we could discuss that in a meeting of course).